tests/e2e/accounts/accounts_users/test_async_accounts_users.py

```
@@ -0,0 +1,200 @@
+    async_mpt_vendor,
+    created_account_user,
+    account_id,
+):
+    """Test resending an invite to an account user."""
+    account_user_data = await created_account_user()
+    users = async_mpt_vendor.accounts.accounts.users(account_id=account_id)
+    resend_invite = await users.resend_invite(account_user_data.id)
+    assert resend_invite is not None
+
+
+def test_account_user_group_post(created_account_user_group):  # noqa: AAA01
```

Ruff (0.14.6)

Unused `noqa` directive (unknown: `AAA01`)

Remove unused `noqa` directive

(RUF100)